model_hub/examples/huggingface/token-classification/README.md

```diff
@@ -0,0 +1,29 @@
+# Token Classification
+This example mirrors the [token-classification example](https://github.com/huggingface/transformers/tree/master/examples/token-classification) from the original huggingface transformers repo.  
+
+## Files
+* **ner_trial.py**: The core code implementing the [Determined trial interface](https://docs.determined.ai/latest/reference/api/pytorch.html#pytorch-trial) for this example.  This is super easy with the [helper functions provided in model-hub](../model_hub/transformers/_trial.py).
```

Comment:
This link should point to BaseTransformerTrial, not the PyTorch Trial interface right?

Reply:
clarified the description for `ner_trial.py`